source/_integrations/flux_led.markdown

```diff
@@ -23,7 +25,69 @@ Examples of controllers:
+
+| Effect Name                                                                                                  | Description                                                        |
+|--------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|
+| `colorloop`                                                                                                  | Smoothly transitions through the rainbow.                          |
+| `colorjump`                                                                                                  | Jumps through seven different rainbow colors.                      |
+| `colorstrobe`                                                                                                | Strobes each rainbow color in a loop.                              |
+| `red_fade`, `green_fade`, `blue_fade`, `yellow_fade`, `cyan_fade`, `purple_fade`, `white_fade`               | Fades between the color as indicated in the effect name and black. |
+| `rg_cross_fade`                                                                                              | Fades between red and green.                                       |
+| `rb_cross_fade`                                                                                              | Fades between red and blue.                                        |
+| `gb_cross_fade`                                                                                              | Fades between green and blue.                                      |
+| `red_strobe`, `green_strobe`, `blue_strobe`, `yellow_strobe`, `cyan_strobe`, `purple_strobe`, `white_strobe` | Strobes the color indicated by the effect name.                    |
+| `random`                                                                                                     | Chooses a random color by selecting random values for R, G, and B. |
+
+
+### Custom Effects - Service `flux_led.set_custom_effect`
+
+The `Flux_LED` integration offers a custom service to enable you to set the lights to a custom light effect. 
+
+| Service data attribute | Description |
+| ---------------------- | ----------- |
+| `entity_id` | The entity_id of the LED light to set the effect on. |
+| `colors` | List of RGB colors to transition between in your effect. (Max 16, Required) |
+| `speed_pct` | The speed of the effect in % (0-100. Default 50) |
+| `transition` | The transition effect you would like. Valid options are `gradual`, `jump`, or `strobe`. (Default `gradual`) |
+
+```yaml
+#Example Service Call
+entity_id: light.led_strip
+colors:
+  - [255,0,0]
+  - [0,255,0]
+  - [0,0,255]
+speed_pct: 80
+transition: jump
+```
+
+### Depreceated configuration.yaml
```

Comment:
Usually we remove the yaml configuration information once its deprecated to prevent users from going down a path that will shortly not exist.

Reply:
I wasn't sure.  I'll remove it.  I also added % include integrations/config_flow.md %